handlers/status_update/__init__.py

```diff
@@ -33,4 +33,9 @@ def on_pr_closed(self, api, payload):
         if payload['pull_request']['merged']:
             api.remove_label("S-awaiting-merge")
 
+    def on_build_status(self, api, payload):
+        if payload['context'] == 'continuous-integration/travis-ci/pr':
+            if payload['state'] == 'failure' or payload['state'] == 'error':
```

Comment:
I assume one of them means an internal error occurred, rather than an error during the build process, but I'm not sure. Probably best to leave both right now.